NuGet.Config

```diff
@@ -10,6 +10,7 @@
       Used for azure-sdk-tools repo until issue https://github.com/Azure/azure-sdk-tools/issues/1329 is addressed
     -->
     <add key="azure-sdk-tools" value="https://azuresdkartifacts.blob.core.windows.net/azure-sdk-tools/index.json" />
+    <add key="DEBUG" value="D:\Git\azure\autorest.csharp\artifacts\packages\Debug" />
```

Comment:
should this be added?

Reply:
no that was accidentally added, I removed it